### PR TITLE
Implement StorageCollection and add StorageCollection On ComputerSystem

### LINKED ISSUE #97: Add storage collection link to ComputerSystem
As oneview-redfish-toolkit user, I access "/redfish/v1/Systems/<server_hardware_uuid>" and receive basic information of the server hardware containing a link to Storage collection.

### LINKED ISSUE #99: Implement StorageCollection
As oneview-redfish-toolkit user, I access "/redfish/v1/Systems/<server_hardware_uuid>/Storage" and receive a list of Storage objects.

### PR DESCRIPTION
**Pull request by danielreed**
Add StorageCollection on ComputerSystem.
Closes #97 and closes #99

### CODE REVIEW

File: oneview_redfish_toolkit/blueprints/storage_collection.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    json_str = storage.serialize()
+
+    return Response(
+        response=json_str,
+        status=status.HTTP_200_OK,
+        mimetype="application/json")
+
+
+@storage_collection.errorhandler(
```
Comment (llvieira) [inline oneview_redfish_toolkit/blueprints/storage_collection.py]: Method not used. If not have any chance to occur an **INTERNAL_SERVER_ERROR** you can remove it.

File: oneview_redfish_toolkit/api/storage_collection.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        Populates self.redfish with some hardcoded StorageCollection
+        values and with the...
+    """
+
+    SCHEMA_NAME = 'StorageCollection'
+
+    def __init__(self, uuid):
+        """StorageCollection constructor
+
+            Populates self.redfish with a hardcoded StorageCollection
```
Comment (ffdarkpenguin) [inline oneview_redfish_toolkit/api/storage_collection.py]: Are we not gonna consider the case of multiple storage in a server-hardware?

File: oneview_redfish_toolkit/api/storage_collection.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+            Args:
+                ...
+        """
+        super().__init__(self.SCHEMA_NAME)
+
+        self.redfish["@odata.type"] = \
+            "#StorageCollection.StorageCollection"
+        self.redfish["Name"] = "Storage Collection"
+        self.redfish["Members@odata.count"] = 1
```
Comment (ffdarkpenguin) [inline oneview_redfish_toolkit/api/storage_collection.py]: Is it OK to be hard-coded to 1??

File: oneview_redfish_toolkit/blueprints/storage_collection.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+@storage_collection.route(
+    "/redfish/v1/Systems/<uuid>/Storage", methods=["GET"])
+def get_storage_collection(uuid):
+    """Get the Redfish Storage Collection.
+
+    Return StorageCollection Redfish JSON.
+    """
+
+    storage = StorageCollection(uuid)
```
Comment (ffdarkpenguin) [inline oneview_redfish_toolkit/blueprints/storage_collection.py]: you are not checking in OV if this UUID is valid.

File: oneview_redfish_toolkit/api/storage_collection.py (hunk 2 above)
Comment (danielreed) [reply oneview_redfish_toolkit/api/storage_collection.py]: The OneView SHT storageCapabilities is an object and not an array so we are going to model this as a collection with a single member.

File: oneview_redfish_toolkit/api/storage_collection.py (hunk 3 above)
Comment (danielreed) [reply oneview_redfish_toolkit/api/storage_collection.py]: Yes I think its okay

File: oneview_redfish_toolkit/blueprints/storage_collection.py (hunk 4 above)
Comment (danielreed) [reply oneview_redfish_toolkit/blueprints/storage_collection.py]: Do we ever check that the UUID is valid in any of the blueprints?  I guess I could check that its not None but not sure how else to check if its valid.

File: oneview_redfish_toolkit/blueprints/storage_collection.py (hunk 1 above)
Comment (danielreed) [reply oneview_redfish_toolkit/blueprints/storage_collection.py]: Ok good point

File: oneview_redfish_toolkit/blueprints/storage_collection.py (hunk 4 above)
Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/blueprints/storage_collection.py]: No, I mean get the SH from oneview to see if this UUID if from a valid SH. In others blueprints we always fetch info from oneview and doing so we validate the UUID. IMO you should get server hardware and return a 404 if the SH is not valid or a 500 if another error occur.

Comment (ricardoas) [reply oneview_redfish_toolkit/blueprints/storage_collection.py]: agreed

**Pull request merged**